eZ/Publish/Core/Persistence/Legacy/Tests/Content/UrlAlias/UrlAliasHandlerTest.php

```diff
@@ -3224,17 +5067,40 @@ protected function getHandler()
         );
     }
 
+    protected function getLanguageHandler()
+    {
+        if (!isset($this->languageHandler)) {
+            $this->languageHandler = new LanguageHandler(
+                new LanguageGateway(
+                    $this->getDatabaseHandler()
+                ),
+                new LanguageMapper()
+            );
+        }
+
+        return $this->languageHandler;
+    }
+
+    protected function getLanguageMaskGenerator()
```

Comment:
Added in https://github.com/ezsystems/ezpublish-kernel/pull/1781/commits/2ec5c0d73ce8d78d41fff616bcdef9610a55718c